xml/System.Reflection.PortableExecutable/PEDirectoriesBuilder.xml

```diff
@@ -17,7 +17,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Builds PE directories.</summary>
```

Comment:
@tmat I added this but I don't think it's descriptive enough. Can you please provide a better summary?